docker/boot_notebook.bash

```diff
@@ -10,7 +10,7 @@ echo "$INFO" "  Workdir :$(pwd)"
 
 # Trust all notebooks in the notebooks folder
 echo "$INFO" "trust all notebooks in path..."
-find "${NOTEBOOK_BASE_DIR}" -name '*.ipynb' -type f -exec jupyter trust {} +
+find "${NOTEBOOK_BASE_DIR}" -name '*.ipynb' -type f | xargs -I % /bin/bash -c 'jupyter trust % || true'
```

Comment:
question: if I get it correctly this will silently fail right? is there no way of outputing the filename into a file like ```errors.txt``` ?

Reply:
actually it prints the output in the terminal. See below. It fails on the first succeeds on the second.

```shell
find "${NOTEBOOK_BASE_DIR}" -name '*.ipynb' -type f  | xargs -I % /bin/bash -c 'jupyter trust % || true'
Traceback (most recent call last):
  File "/opt/conda/lib/python3.9/site-packages/nbformat/reader.py", line 14, in parse_json
    nb_dict = json.loads(s, **kwargs)
  File "/opt/conda/lib/python3.9/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/opt/conda/lib/python3.9/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/opt/conda/lib/python3.9/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/bin/jupyter-trust", line 10, in <module>
    sys.exit(TrustNotebookApp.launch_instance())
  File "/opt/conda/lib/python3.9/site-packages/jupyter_core/application.py", line 264, in launch_instance
    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/traitlets/config/application.py", line 846, in launch_instance
    app.start()
  File "/opt/conda/lib/python3.9/site-packages/nbformat/sign.py", line 606, in start
    self.sign_notebook_file(notebook_path)
  File "/opt/conda/lib/python3.9/site-packages/nbformat/sign.py", line 576, in sign_notebook_file
    nb = read(f, NO_CONVERT)
  File "/opt/conda/lib/python3.9/site-packages/nbformat/__init__.py", line 143, in read
    return reads(buf, as_version, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/nbformat/__init__.py", line 73, in reads
    nb = reader.reads(s, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/nbformat/reader.py", line 58, in reads
    nb_dict = parse_json(s, **kwargs)
  File "/opt/conda/lib/python3.9/site-packages/nbformat/reader.py", line 17, in parse_json
    raise NotJSONError(("Notebook does not appear to be JSON: %r" % s)[:77] + "...") from e
nbformat.reader.NotJSONError: Notebook does not appear to be JSON: ''...
Notebook already signed: /home/jovyan/work/workspace/Untitled.ipynb
```